data/json/terrain.json

```diff
@@ -8854,7 +8854,24 @@
       "sound": "crunch!",
       "sound_fail": "clang!",
       "ter_set": "t_pavement_bg_dp",
-      "items": [ { "item": "scrap", "count": [ 3, 6 ] } ]
+      "items": [ { "item": "pipe", "count": [ 1, 2 ] }, { "item": "scrap", "count": [ 3, 6 ] } ]
+    }
+  },
+  {
+    "type": "terrain",
+    "id": "t_guardrail",
+    "name": "guard rail",
+    "symbol": "#",
+    "color": "light_gray",
+    "move_cost": 3,
+    "flags": [ "TRANSPARENT", "NOITEM", "REDUCE_SCENT", "MOUNTABLE", "SHORT", "THIN_OBSTACLE", "ROAD" ],
```

Comment:
Why `REDUCE_SCENT` flag? It's a short and non-solid rail after all. The same for bridge guard rails.

Reply:
See every other fence or half wall tile. I think the intended effect might be that you can lose pursuit by hopping fences, but that's a complete guess.

Reply:
I agree though that it's probably not intended.